### PR TITLE
Fix for issue #2225

### PR DESCRIPTION
**Pull request by dfk7677**
Added a check for file _users/default.png_ (default avatar image) when deleting image files, as this file is not supposed to be deleted.

This is a fix for [#2225](https://github.com/the-control-group/voyager/issues/2225).

### CODE REVIEW

**Comment by fletch3555** [comment]
Please use feature branches when committing changes to open a PR.  You've now introduced additional changes into this PR (bc2b179).  Please revert these changes as they're unrelated and I can't accept the PR with them

**Comment by dfk7677** [comment]
I am sorry about this. Changes are reverted now.

**Comment by fletch3555** [comment]
Very good, thank you.

**Comment by emptynick** [comment]
I think the filename should not be hardcoded, as it is a config var.
Just use:
`if ($data->{$row->field} != config('voyager.user.default_avatar')) {`

File: src/Http/Controllers/VoyagerBreadController.php [hunk 1]
```diff
@@ -423,7 +423,9 @@ protected function cleanup($dataType, $data)
     public function deleteBreadImages($data, $rows)
     {
         foreach ($rows as $row) {
-            $this->deleteFileIfExists($data->{$row->field});
+            if ($data->{$row->field} != 'users/default.png') {
```
Comment (fletch3555) [inline src/Http/Controllers/VoyagerBreadController.php]: As @emptynick suggested, `'users/default.png'` should not be hardcoded, and should be replaced with the config var.

**Review by fletch3555 — CHANGES_REQUESTED** [review]

**Comment by dfk7677** [comment]
Thanks @emptynick . Change done @fletch3555 .

**Review by fletch3555 — APPROVED** [review]

**Comment by fletch3555** [comment]
Perfect, thanks! @emptynick, have you given this a look from a QA perspective?  If all good, I can merge it

**Comment by emptynick** [comment]
Yes I did, its working fine.
Just trying to think about any cases where this would get a prolem - but I cant think of any.

**Comment by fletch3555** [comment]
Nor can I, thanks!

**Pull request merged**